lex.yy.c missing unistd.h for isatty definition with flex 2.5.37

lex.yy.c is generated without a include for unistd.h, which defines isatty.

    Compiling lex.yy.c
    lex.yy.c: In function ‘yy_init_buffer’:
    lex.yy.c:4154:9: warning: implicit declaration of function ‘isatty’ [-Wimplicit-function-declaration]
             b->yy_is_interactive = file ? (isatty( fileno(file) ) > 0) : 0;
             ^

--- a/cfg.lex
+++ b/cfg.lex
@@ -119,7 +119,7 @@
 	} *oss_yy_fname_list = 0;
 
 	/* hack to solve the duplicate declaration of 'isatty' function */
-#if YY_FLEX_MAJOR_VERSION <= 2 && YY_FLEX_MINOR_VERSION <= 5 && YY_FLEX_SUBMINOR_VERSION <= 36
+#if YY_FLEX_MAJOR_VERSION <= 2 && YY_FLEX_MINOR_VERSION <= 5 && YY_FLEX_SUBMINOR_VERSION < 36
 	#define YY_NO_UNISTD_H
 #else
 	#include <unistd.h>

--- a/cfg.lex
+++ b/cfg.lex
@@ -119,7 +119,7 @@
 	} *oss_yy_fname_list = 0;
 
 	/* hack to solve the duplicate declaration of 'isatty' function */
-#if YY_FLEX_MAJOR_VERSION <= 2 && YY_FLEX_MINOR_VERSION <= 5 && YY_FLEX_SUBMINOR_VERSION <= 38
+#if YY_FLEX_MAJOR_VERSION <= 2 && YY_FLEX_MINOR_VERSION <= 5 && YY_FLEX_SUBMINOR_VERSION <= 36
 	#define YY_NO_UNISTD_H
 #else
 	#include <unistd.h>

